tensorflow/tools/pip_package/BUILD

```diff
@@ -169,6 +169,19 @@ sh_binary(
         "//tensorflow:windows_msvc": [":simple_console_for_windows"],
         "//conditions:default": COMMON_PIP_DEPS + [
             ":simple_console",
+            "//tensorflow:tensorflow_py",
```

Comment:
I cannot see where this change is coming from.
These seem to be causing the build failures.

@yifeif could you take a look?